pkg/sessionctx/variable/sysvar.go

```diff
@@ -2965,6 +2966,40 @@ var defaultSysVars = []*SysVar{
 		}},
 }
 
+func GlobalSyetemVariableInitialValue(varName, varVal string) string {
```

Comment:
Can we add a unittest for this?

Reply:
```suggestion
// GlobalSystemVariableInitialValue gets the default value for a system variable including ones that are dynamically set (e.g. based on the store)
func GlobalSystemVariableInitialValue(varName, varVal string) string {
```
Please note the typo in the function name.

Reply:
> Can we add a unittest for this?

@dveeden Thank you for your suggestions. Can you explain more about the unittests，is it about to check the initalValue of some global variables under the `InTest`?

Other suggestions have been updated.

Reply:
> Can we add a unittest for this?

Added [0bdec4e](https://github.com/pingcap/tidb/pull/49524/commits/0bdec4e799b38b6ed2b0dae66d4e3a962aed4e14)